packages/contracts/contracts/minter-suite/Minters/MinterRAMV0.sol

```diff
@@ -1185,8 +1185,37 @@ contract MinterRAMV0 is ReentrancyGuard, ISharedMinterV0, ISharedMinterRAMV0 {
     }
 
     /**
-     * Returns balance of project `projectId` on core contract `coreContract`
-     * on this minter contract.
+     * @notice Convenience view function that returns the bid value associated
+     * with a given slot index for the specified project's auction, in Wei.
+     * Reverts if the slot index is out of range (greater than 511).
+     * Reverts if the project's auction is not configured.
+     * @param projectId Project ID to get the bid value for
+     * @param coreContract Core contract address for the given project
+     * @param slotIndex Slot index to get the bid value for
+     */
+    function slotIndexToBidValue(
```

Comment:
new function does not require audit, and is for convenience only (may be removed due to bytecode size limit)